lib/asciidoctor-revealjs/converter.rb

```diff
@@ -0,0 +1,30 @@
+require 'asciidoctor/converter/html5'
+require 'asciidoctor/converter/composite'
+require 'asciidoctor/converter/template'
+
+module Asciidoctor
+module Revealjs
```

Comment:
I've recently discovered a trick that allows you to get the scoping behavior, but write this on a single line:

```ruby
module Asciidoctor; module Revealjs
  ...
end; end
```

That differs from the following where the scope of the intermediate modules are not visible:

```
module Asciidoctor::Revealjs
  ...
end
```